suricata.yaml.in

```diff
@@ -1424,6 +1424,7 @@ flow:
   emergency-recovery: 30
   #managers: 1 # default to one flow manager
   #recyclers: 1 # default to one flow recycler thread
+  force-reuse: 1 # Default to forcing flow reuse in low memory conditions
```

Comment:
we can enable it by default in the code, and leave it commented out here

additionally, it should use `true/false` instead of suggesting it is a numeric value